docs/user/field-selectors.md

```diff
@@ -15,7 +15,8 @@ Each resource supports the following metadata fields:
 - `metadata.annotations`
 - `metadata.creationTimestamp`
 
-> ![NOTE] While `metadata.labels` can be selected using Field Selectors, for more extensive label querying options, consider using Label Selectors.
+> [!NOTE]
```

Comment:
@fzdarsky had to fix it according to the following:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts